docs/team/eugene-ong-w-x.md

```diff
@@ -5,39 +5,36 @@ title: Eugene Ong Wei Xiang's Project Portfolio Page
 
 ### Project: SoConnect
 
-SoConnect is a desktop app for NUS SoC students to manage their contacts, optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+SoConnect is a **contact management and task management desktop app** that aims to help NUS SoC students stay better connected to their school life, both in terms of social connections and school tasks. To better cater to NUS SoC students, SoConnect is **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). SoConnect contains 42 kLoC written in Java and uses JavaFX.
 
-Given below are my contributions to the project.
+Given below are my contributions to SoConnect.
 
-* **New Feature**: Added the ability to undo/redo previous commands. (remove)
-    * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-    * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-    * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to sort a list of contacts.
```

Comment:
I think we are supposed to write the feature itself like `Sort Feature`

Reply:
Ok good idea